content/docs/start/experiments.md

```diff
@@ -2,293 +2,225 @@
-Queued experiment 'd3f6d1e' for future execution.
-$ dvc exp run --queue -S train.min_split=64
-Queued experiment 'f1810e0' for future execution.
-$ dvc exp run --queue -S train.min_split=2 -S train.n_est=100
-Queued experiment '7323ea2' for future execution.
-$ dvc exp run --queue -S train.min_split=8 -S train.n_est=100
-Queued experiment 'c605382' for future execution.
-$ dvc exp run --queue -S train.min_split=64 -S train.n_est=100
-Queued experiment '0cdee86' for future execution.
+$ dvc pull
 ```
 
-Next, run all (`--run-all`) queued experiments in parallel (using `--jobs`):
+The repository already contains the necessary configuration to run the
+experiments.
 
-```dvc
-$ dvc exp run --run-all --jobs 2
-```
+</details>
 
-## Comparing many experiments
+## Running the experiment with default parameters
 
-To compare all of these experiments, we need more than `diff`. `dvc exp show`
-compares any number of experiments in one table:
+The purpose of `dvc exp` subcommands is to run the experiments without
+committing parameter and dependency changes to Git. The artifacts like models,
+metrics produced by each experiment are tracked by DVC and persisted on demand.
 
-```dvc
-$ dvc exp show --no-timestamp \
-               --include-params train.n_est,train.min_split
-```
+Running the experiment with the default project settings requires only the
+command:
 
-```dvctable
-┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ neutral:**Experiment**    ┃ metric:**avg_prec** ┃ metric:**roc_auc** ┃ param:**train.n_est**┃ param:**train.min_split** ┃
-┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
-│ **workspace**     │  **0.56191** │ **0.93345** │ **50**         │ **2**               │
-│ **master**        │  **0.55259** │ **0.91536** │ **50**         │ **2**               │
-│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
-│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64              │
-│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2               │
-│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8               │
-│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64              │
-│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
-└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-b28f0
+Experiment results have been applied to your workspace.
+...
 ```
 
-Each experiment is given an arbitrary name by default (although we can specify
-one with `dvc exp run -n`.) We can see that `exp-98a96` performed best among
-both of our metrics, with 100 estimators and a minimum of 64 samples to split a
-node.
-
-> See `dvc exp show --help` for more info on its options.
-
-## Persisting experiments
-
-Now that we know the best parameters, let's keep that experiment and ignore the
-rest.
+It runs the specified command (`python train.py`) in `dvc.yaml`. That command
```

Comment:
I would put even all this stuff after the `dvc exp show` - it explains how (deep tech details), we need to show the idea from the user perspective (and use expandable sections to go deep if needed).

Reply:
I think it's better to present `dvc exp show` after a few experiments. To run _a few_, we need to specify the parameters. I'll move the `show` section after _run by setting params_.